cartography/data/ontology/users.json

```diff
@@ -0,0 +1,129 @@
+{
+    "anthropic": {
+        "nodes": {
+            "AnthropicUser": {
+
```

Comment:
```suggestion
```
lets generally get rid of all the extra blank lines in the json, perhaps we should add a json formatter to the linter?